packages/gafl-webapp-service/src/uri.js

```diff
@@ -48,6 +48,7 @@ export const RENEWAL_INACTIVE = { uri: '/buy/renew/inactive', page: 'renewal-ina
 export const AUTHENTICATE = { uri: '/buy/renew/authenticate' }
 export const RENEWAL_START_DATE = { uri: '/buy/renew/renewal-start-date', page: 'renewal-start-date' }
 export const RENEWAL_START_VALIDATE = { uri: '/buy/renew/renewal-start-date/validate' }
+export const RENEWAL_LICENCE = { uri: '/renew-my-licence/{reference}', page: '/attribution?utm_campaign=renewals&urm_source=aen_invitation&reference=<licence number>' }
```

Comment:
page shouldn't be defined like this - if you look, no other uri constant is set up like that. Also, unless I've missed something, you don't actually use the page property. I think you meant to, see my comment on your test, but it should either be setup in the journey definition or by using `ATTRIBUTION` above and building a query string.

Reply:
ready for re review